.travis.yml

```diff
@@ -8,7 +8,7 @@ python:
   - "3.4"
   - "3.5"
   - "3.6"
-  - "3.6-dev"
+  - "3.7-dev"
```

Comment:
This sounds like the dev branch of 3.7, not 3.7 final.

Reply:
I think 3.7-dev is the latest version when without specifying dist at current.

- https://github.com/travis-ci/travis-ci/issues/9815
- https://github.com/rq/rq/pull/975#issuecomment-402719107